colibre/auto_plotter/mass_functions.yml

```diff
@@ -116,6 +116,7 @@ stellar_mass_function_50:
   type: "massfunction"
```

Comment:
Note that here the type of the plot is `massfunction` but in `correction.yml`, it is `mass_function`. Shouldn't these two be the same?

Reply:
No, because they are actually not the same thing. `massfunction` is a plot type, while `mass_function` is a line type. That is what the autoplotter uses internally. I don't know if that distinction is useful.

If we would use `massfunction`, then the `velociraptor-compute-box-size-correction` script would not be able to find the line in the pipeline `data*.yml` output file. I could hardcode a conversion `mass_function-->massfunction` into the script if you think that is useful.